NOTES.md

```diff
@@ -1,3 +1,19 @@
+# Release 2.1.4 (Forbidden Planet)
+
+Released on 2015-10-15
+
+Bug fix release
+
+* Fixed a second data corruption bug in the b-tree implementation (#4769)
```

Comment:
I wrote "second" because it's the same issue number as a fix with the same description in the 2.1.3 release.
I'll try to add more details instead to distinguish it.